packages/devextreme/js/__internal/ui/splitter/splitter.ts

```diff
@@ -75,7 +76,11 @@ class Splitter extends (CollectionWidget as any) {
 
     this._toggleOrientationClass();
 
-    this._layout = getInitialLayout(this.option('items'));
+    const items = this.option('items');
+
+    const elementSize = getElementSize(this.$element(), items, this.option('orientation'), this.option('width'), this.option('height'));
```

Comment:
This code is not working in case: 
- Splitter are located inside TabPanel content. 
- Templates are render asynchroniously
- May be after Render in invisible container